### ISSUE TITLE
Invalid card is not recognized as one at subscription

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla/fxa#5877]
#### Affected versions:
- All Firefox;

#### Affected FxA versions:
- 1.178.0;

#### Affected Platforms:
- Windows 10 (64-bit);

#### Prerequisites:
- Have and active FxA account;

#### Steps to reproduce
1. Load the VPN main page https://stage-vpn.guardian.nonprod.cloudops.mozgcp.net/ 
2. Click on the “Subscribe Now” button;
2. Enter the following card no: 2123 2123 1323 3123 ;
4. Fill all fields and click “Pay Now” button;

#### Expected result:
- While clicking outside the card field or on the “Pay now” button an “invalid card” message is displayed under the card field.

#### Actual result:
- The message “Something went wrong. Please try again later.” is displayed and 402 error in the console.

#### Notes:
- If the same card number is used to update the payment details, the “basic-error-message” message is displayed
![unexistentcard-noerror](https://user-images.githubusercontent.com/15685960/86795222-06254080-c076-11ea-85d5-cab51eeaa237.png)

![image](https://user-images.githubusercontent.com/15685960/86795209-03c2e680-c076-11ea-9051-a947917162c4.png)

**Comment by data-sync-user** [comment]
➤ Wil Clouser commented:

What is that card number?  Some card numbers are built to create errors on purpose, but I don't see that number on the Stripe test card page @ https://stripe.com/docs/testing#cards

**Comment by ValentinaPC** [comment]
> 
> 
> ➤ Wil Clouser commented:
> 
> What is that card number?  Some card numbers are built to create errors on purpose, but I don't see that number on the Stripe test card page @ https://stripe.com/docs/testing#cards

Yeah, is not in there. I've randomly entered a bunch of numbers and was expecting to see the invalid card error, in stead I was surprised with other 2 messages. Was a perfect synchronization. :)
